plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.test.tsx

```diff
@@ -20,6 +20,11 @@ import { fireEvent, screen } from '@testing-library/react';
 import * as React from 'react';
 import { EntityContextMenu } from './EntityContextMenu';
 
+jest.mock('@backstage/plugin-catalog-react', () => ({
```

Comment:
This feels like a very targeted bit of mocking which might make the tests a little more brittle in the future. Did you explore the possibility of providing the MockPermissionApi in the test app used in this suite instead?

Reply:
Good catch, fixed!